### DISCUSSION TITLE
Cookies shared between subdomains

### DISCUSSION THREAD

**Discussion by Hideman85** [vercel/vercel#7907] [Help]
Hi,

I have the issue that I need to share the cookie along the sub-domains generated by vercel, I tried to set the domain to `vercel.app` but this is forbidden as part of the [Mozilla Public List](https://publicsuffix.org/).
There is also plenty of topics with the same issue https://github.com/vercel/vercel/discussions/6060 https://github.com/vercel/vercel/discussions/4235 https://github.com/vercel/vercel/discussions/5290 for citing a few of them.

I'm wondering if vercel could create a subdomain in a way we could set cookies correctly, something like `deployment.team.vercel.app` then we could set the cookies to domain `team.vercel.app` and this should be of course allowed.

I was also looking the doc about custom domains and wildcards :thinking: can I setup a custom domain in the form of `deployment.vercel.mydomain.com` so any commit/branch/permission would use the custom subdomain and so solve the cookies issue too?

I'm looking forward to some help there :slightly_smiling_face:

**Comment by aakash14goplani** [comment]
@Hideman85 - were you able to find any solution for this?

**Reply by Hideman85** [reply]
Wow, it's been a while, actually nope and we stopped using vercel so we did not go forward with that. From what I recall back that time there is no way to bypass this browser security actually. Neither we were been able to workaround it 😕
